### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dockercore**
feat: Add upgrade instructions for Docker deployment
﻿
﻿
﻿
Detailed steps for stopping the service, backing up data, and upgrading to a new version
﻿
Includes commands for backing up Docker image ID, database files, and docker-compose.yaml
﻿
Provides an example of downloading a specific version (0.6.5) of the docker-compose.yaml file
﻿
﻿
﻿
﻿
﻿
﻿
﻿
## Type of Change
﻿
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
﻿
﻿
# Suggested Checklist:
﻿
- [ ] `optional` I have made corresponding changes to the documentation 
﻿
﻿

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,6 +183,39 @@ After running, you can access the Dify dashboard in your browser at [http://loca
 
 > If you'd like to contribute to Dify or do additional development, refer to our [guide to deploying from source code](https://docs.dify.ai/getting-started/install-self-hosted/local-source-code)
 
+
+##  Upgrade steps
+- 1. Stop the service，Command, please execute in the docker directory
+```bash
+docker-compose down
```
Comment (crazywoola) [inline README.md]: Please use `docker compose` as docker docs suggests.

**Review by crazywoola — CHANGES_REQUESTED** [review]
And please update other languages accordingly.

File: README.md (hunk 1 above)
Comment (dockercore) [reply README.md]: OK

Comment (crazywoola) [inline README.md]:
```suggestion
docker compose down
```

File: README.md [hunk 2]
```diff
@@ -183,6 +183,39 @@ After running, you can access the Dify dashboard in your browser at [http://loca
...
+  ```
+
+- 5. Download the new version of the docker-compose YAML file，using version 0.6.5 as an example
+```bash
+wget -O docker-compose.yaml https://github.com/langgenius/dify/raw/0.6.5/docker/docker-compose.yaml
+```
+
+- 6. Upgrade completed
+```bash
+ docker-compose up -d
```
Comment (crazywoola) [inline README.md]:
```suggestion
 docker compose up -d
```

**Review by crazywoola — DISMISSED** [review]

**Review by crazywoola — APPROVED** [review]

**Comment by guchenhe** [comment]
README might not be the best place for this, would you open a PR to put this in our docs instead? https://docs.dify.ai/getting-started/install-self-hosted/docker-compose

I think this could be an additional section in https://docs.dify.ai/getting-started/install-self-hosted/docker-compose, under `Upgrade Dify`

**Comment by crazywoola** [comment]
Close for now.


> README might not be the best place for this, would you open a PR to put this in our docs instead? https://docs.dify.ai/getting-started/install-self-hosted/docker-compose
> 
> I think this could be an additional section in https://docs.dify.ai/getting-started/install-self-hosted/docker-compose, under `Upgrade Dify`

You can open a pr for this doc.

**Pull request closed without merging**